Documentation/ApiOverview/Typo3CoreEngine/Database/Index.rst

```diff
@@ -193,6 +193,38 @@ Command keywords and values
          make a copy of the record on the same page.
 
 
+ - :Command:
+         copyToLanguage
+   :Type:
+         integer
+   :Value:
+         A new DataHandler command 'copyToLanguage' has been introduced. It behaves like 'localize' command
+         (both record and child records are copied to given language), but does not set transOrigPointerField fields (e.g. l10n_parent).
+
+         The 'copyToLanguage' command should be used when localizing records in the "Free Mode". This command is used when localizing
+         content elements using translation wizard's "Copy" strategy.
+
+         The 'localize' DataHandler command should be used when translating records in "Connected Mode" (strict translation of records from the default language).
+         This command is used when selecting "Translate" strategy in content elements translation wizard.
+
+
+ - :Command:
+         inlineLocalizeSynchronize
+   :Type:
+         array
+   :Value:
+         Performs localization or synchronization of child records.
+         The command structure is like::
+
+             [parent][13][inlineLocalizeSynchronize] = [
+               field: name of the parent field,
+               language: id of the target language,
+               action: either "localize" or "synchronize",
+               ids: array of child-ids to be localized, e.g. [1, 2, 3]
+             ]
+
+         Either "action" or "ids" must be defined.
```

Comment:
Better add a concrete example with some sample values here and explain by referring to the numbers.